app/models/interaction.rb

```diff
@@ -7,4 +7,20 @@ class Interaction < ActiveRecord::Base
 
   validates :category, presence: true
   validates :response, presence: true
+  validate :one_of_three_interactions
+
+  def one_of_three_interactions
```

Comment:
Assignment Branch Condition size for one_of_three_interactions is too high. [33.84/15]